암호 지정시 두 암호가 틀릴경우 암호 지정 첫화면으로 돌아가도록 변경했습니다.

intmain opened the pull request:
1. 암호화면에서는 오리엔테이션을 막았습니다. 
2. 암호 지정시 두 암호가 틀릴경우 암호 지정 첫화면으로 돌아가도록 변경했습니다.
3. Mode가 New나 Change일 경우에도 작동이 완료되고나면 Delegate 메소드에서 Mode에 접근시 Normal로 변경되어 있어  Mode값을 유지하도록 변경했습니다. 

풀리퀘 받아주시면 감사드리겠습니다.

Commit:
뷰 회전 잠금.

intmain committed:
암호 지정중 암호 확인이 틀렸을 경우엔  암호 지정 첫 화면으로 다시 돌아가도록 변경.

tiny2n commented:
관심갖아 주셔서 감사합니다.
1. 해당 현상 확인하였습니다. 수정 감사합니다.
2. 1. 암호 지정시 두 암호가 틀릴 경우 첫 입력 모드로 돌아가긴 하나 타이틀은 Confirm으로 출력됩니다.
3. 2. 암호 지정 시 첫번째 암호를 신규 암호로, 두번째 암호를 기존암호로 입력하면 Lock이 풀려버리는 버그가 있습니다.
4. Mode 유지 수정 감사합니다.

다시 한번 관심 갖아주셔서 감사합니다. 번거로우시더라도 2-1. 2-2.를 수정해주시면 풀리퀘 해드리겠습니다.

intmain committed:
아래 두가지 버그 수정.
1. 암호 지정시 두 암호가 틀릴 경우 첫 입력 모드로 돌아가긴 하나 타이틀은 Confirm으로 출력됩니다.
2. 암호 지정 시 첫번째 암호를 신규 암호로, 두번째 암호를 기존암호로 입력하면 Lock이 풀려버리는 버그가 있습니다.
